Sources/SwiftDocC/Infrastructure/External Data/OutOfProcessReferenceResolver.swift

```diff
@@ -155,21 +155,29 @@ public class OutOfProcessReferenceResolver: ExternalReferenceResolver, FallbackR
             fatalError("A topic reference that has already been resolved should always exist in the cache.")
         }
         
-        let kind = DocumentationNode.Kind(name: resolvedInformation.kind.name, id: resolvedInformation.kind.id, isSymbol: resolvedInformation.kind.isSymbol)
-        
         let maybeSymbol = OutOfProcessReferenceResolver.symbolSemantic(
-            kind: kind,
+            kind: resolvedInformation.kind,
+            language: resolvedInformation.language,
             title: resolvedInformation.title,
+            abstract: resolvedInformation.abstract,
             declarationFragments: resolvedInformation.declarationFragments,
-            platforms: resolvedInformation.platforms
+            platforms: resolvedInformation.platforms,
+            variants: resolvedInformation.variants
         )
         
+        let name: DocumentationNode.Name
+        if maybeSymbol != nil {
+            name = .symbol(declaration: .init([.plain(resolvedInformation.title)]))
+        } else {
+            name = .conceptual(title: resolvedInformation.title)
+        }
+        
         return DocumentationNode(
             reference: reference,
-            kind: kind,
+            kind: resolvedInformation.kind,
             sourceLanguage: .init(name: resolvedInformation.language.name, id: resolvedInformation.language.id),
             availableSourceLanguages: Set(resolvedInformation.availableLanguages.map { .init(name: $0.name, id: $0.id) }),
-            name: .conceptual(title: resolvedInformation.title),
+            name: name,
```

Comment:
Why is this change in behavior needed here? It doesn't seem incorrect to me; I'm just not seeing how its related to multi-language support.

Reply:
If the node has a conceptual name then its variant symbol names won't be considered. See [this code in LinkTitleResolver.title(for:)](https://github.com/apple/swift-docc/blob/main/Sources/SwiftDocC/Model/Rendering/LinkTitleResolver.swift#L47-L49).

This change is a general bug fix but the issue only surfaces with multi-langue content.

Reply:
Ah got it. Makes sense- thank you for clarifying!